test-suite/run_djinni.sh

```diff
@@ -95,6 +96,7 @@ fi
 "$base_dir/../src/run-assume-built" \
     --java-out "$temp_out_relative/java" \
     --java-package $java_package \
+    --java-generate-interfaces true \
     --java-nullable-annotation "javax.annotation.CheckForNull" \
     --java-nonnull-annotation "javax.annotation.Nonnull" \
     --java-use-final-for-record false \
```

Comment:
Need to enable Java interfaces on the 3rd Djinni run below (GitHub won't let me comment on a line which isn't changed), so that all generation is consistent.  I noticed in looking at diff since my last review that ExternalInterface1.java had reverted back to using `abstract class`.